chapters/substitution.adoc

```diff
@@ -2,13 +2,25 @@
 .Definition
 Robots are substituted by the <<Robot Handler, robot handler>> of the respective team. No other team member is allowed to take robots out or put robots in.
 
-. A <<Robot Handler, robot handler>> who's team is marked for robot substitution performs at least one of the following operations in the given order.
-.. Put robots in. The maximum allowed number of robots of the team on the field must not be exceeded.
-.. Take robots out. If a robot that is taken out is closer than 1 meter to the intersection of the halfway line with one of the touch lines, another robot can be put in right away. Otherwise the robot handler has to wait until the next substitution opportunity.
+. The <<Robot Handler, robot handler>> should prefer to use long sleeves and colors that won't interfere with the vision system.
+
+. Robots can always be substituted during game play if the following specifications are respected:
+
+.. The robot is at least partially inside the <<Field Surface, field margin>>
```

Comment:
General note: It seems that most of the bullet point lists end with a dot. It would be nice to keep consistency.